### ISSUE TITLE
Tab will set focus to layers and arrow keys now nudge layers instead of scrolling the list

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#1682]
Open a doc with multiple layers, 
select a layer, 
cmd/ctrl + f and expand the list, 
use up/down arrow to nav list, 
hit Tab, 
arrow up/down

Result: arrow keys no longer scroll the list and instead nudge the layer

**Comment by iwehrman** [comment]
Since the floating search component is modal, focus should **never** leave the modal.

**Comment by iwehrman** [comment]
FBNC to @chadrolfs.

**Comment by chadrolfs** [comment]
The layer is no longer nudged but arrow keys cease to work in datalist after hitting Tab.  Separate issue or blocked by #1704 or #1705?  If arrow keys aren't supposed to work after hitting Tab and this is as design, re-assign to me or just close.  thanks!

**Comment by chadrolfs** [comment]
I can get into a state (on Windows only) where focus is not set on the Layer Search dialog on initial open (no active cursor in field).  Hitting Tab cycles through the layers, arrows nudge them, etc.

**Comment by iwehrman** [comment]
FBNC to @chadrolfs.
